packages/gatsby-transformer-webapi/scripts/dump.debug.js

```diff
@@ -0,0 +1,59 @@
+const webapi = require('webapi-parser');
```

Comment:
I don't think this file should be included in this merge, for similar reasons as the files it actually dumps.

Reply:
Please propose a better way to manage debugging and development helper code that is necessary to develop on a piece of software. 

It's not practically possible to run the full gatsby in debug mode to interactively explore the data stuctures of the internal API provided by the parser (it's very very slow and does not behave stable).  And the parser's API is very badly documented. 

IMHO  `/scripts` is the conventional place for helper scripts that you need in the development workflow.   Tell me a better place to manage such code an I'm all fine.  But deleting or not sharing it in the team is not an option.

Reply:
For me, if I'm not using the IDE's debugger then I'm logging.

I also gave up on vs code debug configs because of its inconsistencies, but I found that the single file visual studio code debugger (just pressing f5) is the most consistent so I use that quite often.

Reply:
Let's talk about the best debug setup tomorrow in our call and document / commit the best practice - in this kind of RAML parser / data structure debugging code it's such a massive productivity difference to not have to work with logging.  I did spend ages on the old parser until I wrote myself a script like this which works reliably in the  debugger.  About 5* as fast to read the data structures out or the parser API then because you interactively browse all its function calls and directly see the return values.